plugins/base/src/main/kotlin/parsers/MarkdownParser.kt

```diff
@@ -465,46 +466,46 @@ class MarkdownParser(
             DocumentationNode(
                 (listOf(kDocTag) + getAllKDocTags(findParent(kDocTag))).map {
                     when (it.knownTag) {
-                        null -> if (it.name == null) Description(parseStringToDocNode(it.getContent())) else CustomTagWrapper(
-                            parseStringToDocNode(it.getContent()),
+                        null -> if (it.name == null) Description(parseStringToDocNode(it.getContent()) as RootDocTag) else CustomTagWrapper(
```

Comment:
Is it even possible for `parseStringToDocNode` to return something other than `RootDocTag`. If no move cast into that function. If this is possible consider slightly modifying tha api.

Reply:
Yeah, I shouldn't have left it that way, will refactor soon